src/components/graphs/UsageByMonthGraph.tsx

```diff
@@ -0,0 +1,204 @@
+
+                return { month, data: task_usage_hours ?? 0 };
+            }
+        );
+
+        return { data: data_series, hours: hours_series };
+    }, [billingHistory, intl, today]);
+
+    useEffect(() => {
+        if (billingStoreHydrated && billingHistory.length > 0) {
+            if (!myChart) {
+                echarts.use([
+                    GridComponent,
+                    BarChart,
+                    CanvasRenderer,
+                    UniversalTransition,
+                    MarkLineComponent,
+                    TooltipComponent,
+                    LegendComponent,
+                ]);
+
+                const chartDom = document.getElementById(chartContainerId);
+
+                setMyChart(chartDom && echarts.init(chartDom));
+            }
+
+            window.addEventListener('resize', () => {
+                myChart?.resize();
+            });
+
+            const option = {
+                xAxis: {
+                    type: 'category',
+                    data: months,
+                    axisPointer: { type: 'shadow' },
+                },
+                yAxis: [
+                    {
+                        type: 'value',
+                        name: '',
+                        axisLabel: {
+                            formatter: '{value} GB',
+                        },
+                        minInterval: 0.001,
+                        splitLine: {
+                            show: false,
+                        },
+                    },
+                    {
+                        type: 'value',
+                        name: '',
+                        axisLabel: {
+                            formatter: '{value} Hours',
+                        },
+                        minInterval: 0.001,
+                    },
+                ],
+                series: [
+                    {
+                        name: 'Data',
+                        type: 'bar',
+                        barMinHeight: 3,
+                        data: seriesConfigs.data.map(({ month, data }) => [
+                            month,
+                            data.toFixed(3),
+                        ]),
+                        tooltip: {
+                            valueFormatter: (value: number) => {
+                                return `${value} GB`;
+                            },
+                        },
+                    },
+                    {
+                        name: 'Hours',
+                        type: 'bar',
+                        stack: 'Task Hours',
+                        barMinHeight: 3,
+                        data: seriesConfigs.hours.map(({ month, data }) => [
+                            month,
+                            data.toFixed(3),
+                        ]),
+                        tooltip: {
+                            valueFormatter: (value: number) => {
+                                return `${value} Hours`;
```

Comment:
All the content that is hard coded needs to be moved to a lang file.

Reply:
Does that include the `axisLabel` stuff? I wasn't sure about what the right way to put that in the lang file, and I saw it hard-coded elsewhere. Thoughts?

Reply:
I ended up referencing  `intl.messages['admin.billing.graph.usageByMonth.hoursFormatter']` here, and it does work but only coincidentally because `react-intl` and ECharts both use `{value}` to indicate a placeholder

Reply:
We ended up flipping this back cause the original is slightly better.